src/main/java/meditracker/argument/RepeatArgument.java

```diff
@@ -0,0 +1,22 @@
+package meditracker.argument;
+
+/**
+ * Represents the repeat frequency of medication.
+ * Extends the Argument class.
+ */
+public class RepeatArgument extends Argument {
+
+    /**
+     * Constructs a RepeatArgument object with the specified optional status.
+     * @param isOptional true if the argument is optional, false otherwise.
+     */
+    public RepeatArgument(boolean isOptional) {
+        super(
+             ArgumentName.REPEAT,
+                "-rep",
+                "How often do you need to take this medication?",
+                "How often to take medication (eg: once a day, once every week)",
```

Comment:
"eg: once a day" can be misleading, u can maybe put "everyday, every two days, etc."

Reply:
@e0958902 Perhaps you can take a look into this and update accordingly later. Maybe also add a new enum class for these as well, such as `DAILY`, `WEEKLY`, `BIWEEKLY` etc. when you store them within `Medication` class.

Reply:
Oh I think we can just scope within a week, like in a 7 days context. No need biweekly.

Reply:
I would prefer to keep it simple first before we expand into BIWEEKLY and etc.